x-pack/plugins/ml/public/application/components/job_spaces_repair/repair_results.tsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { EuiText, EuiTitle, EuiAccordion, EuiTextColor, EuiHorizontalRule } from '@elastic/eui';
+
+import { RepairSavedObjectResponse } from '../../../../common/types/saved_objects';
+
+export const RepairList: FC<{ repairItems: RepairSavedObjectResponse | null }> = ({
+  repairItems,
+}) => {
+  if (repairItems === null) {
+    return null;
+  }
+
+  return (
+    <>
+      <SavedObjectsCreated repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+
+      <SavedObjectsDeleted repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+
+      <DatafeedsAdded repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+
+      <DatafeedsRemoved repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+    </>
+  );
+};
+
+const SavedObjectsCreated: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.savedObjectsCreated);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsCreated.title"
+              defaultMessage="Missing job saved objects ({count})"
```

Comment:
Per other suggestion, I think we should drop "job":
```suggestion
              defaultMessage="Missing saved objects ({count})"
```
Alternatively, I'd suggest "Missing saved objects for machine learning jobs...." or "Missing X saved objects with type Y", but those seem unnecessarily verbose IMO

Reply:
updated in https://github.com/elastic/kibana/pull/83320/commits/29c6175bd0c1efa92d388f57dfb34a8fcb23e47e